packaging/install

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This is the previously mentioned script to help updating packaging/ in a project.

Remove this file here.